tests/hilti/codegen/type-info.hlt

```diff
@@ -0,0 +1,537 @@
+# @TEST-EXEC: spicy-build -d -S %INPUT type-info.cc && ./a.out
```

Comment:
With you writing code like this I should again look into why I didn't see compiler warnings emitted during JIT :)